Dockerfile

```diff
@@ -10,9 +10,9 @@ FROM php:8.2-cli-alpine
 COPY . /opt/guides
 WORKDIR /opt/guides
 
-COPY --from=Builder /opt/guides/.Build /opt/guides/.Build
-RUN cp guides.xml .Build/guides.xml
+COPY --from=Builder /opt/guides/vendor /opt/guides/vendor
+RUN cp guides.xml guides.xml
```

Comment:
@jaapio Is the the correct path then? `.Build/guides.xml` -> `guides.xml`

Reply:
yes that is correct

Reply:
however we do not have to copy it then. The guides always searches in the parent directory of vendor

Reply:
Yeah, we can simply remove this cp command

Reply:
Done